eclair-core/src/main/scala/fr/acinq/eclair/payment/PaymentRequest.scala

```diff
@@ -200,20 +200,45 @@ object PaymentRequest {
   }
 
   /**
-    * Routing Info Tag
+    * Hidden hop
     *
     * @param pubkey          node id
     * @param channelId       channel id
     * @param fee             node fee
     * @param cltvExpiryDelta node cltv expiry delta
     */
-  case class RoutingInfoTag(pubkey: PublicKey, channelId: BinaryData, fee: Long, cltvExpiryDelta: Int) extends Tag {
+  case class HiddenHop(pubkey: PublicKey, channelId: BinaryData, fee: Long, cltvExpiryDelta: Int) {
+    def pack: Seq[Int5] = pubkey.toBin ++ channelId ++ Protocol.writeUInt64(fee, ByteOrder.BIG_ENDIAN) ++
```

Comment:
the type here is Byte, not Int5 (they're just aliases but it's good for readability)